Formula/f/fw.rb

```diff
@@ -5,6 +5,11 @@ class Fw < Formula
   sha256 "9b87924c5384c65f7012ae545d40e550f830fdf1b3e75bbbde5f9b4bd64aab86"
   license "WTFPL"
 
+  livecheck do
```

Comment:
```suggestion
  # This repository also contains version tags for other tools (e.g., `v4.4.0`
  # is an `fblog` tag), so we can't reliably determine which tags are for `fw`.
  # Upstream only creates GitHub releases from `fw` tags, so we have to check
  # releases instead.
  livecheck do
```

The `stable` URL is a tag archive, so we should explain why we have to use `GithubLatest` here.